.storybook/stories/DataFiltering/components/DataTable.tsx

```diff
@@ -150,12 +150,7 @@ export const DataTable = forwardRef<HTMLTableElement, DataTableProps>(
 											);
 										}
 										return (
-											<TableHeader
-												key={sortKey}
-												scope="col"
-												textAlign={textAlign}
```

Comment:
Dates shouldn't been aligned to right but wasn't picked up since it didn't work before the recent fix